3_javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,77 @@
+//Voice assistant
+let personName = "";
+let toDoList = [];
+
+function getReply(command) {
+  if (command.startsWith("Hello my name is")) {
+    personName = command.split(" ")[4];
+    console.log(`nice to meet you ${personName}`);
+    return;
+  }
+
+  if (command === "What is my name") {
+    if (personName === "") console.log("I dont have your name");
```

Comment:
Omitting the curly braces for this `if`-statement is valid, but it can be tricky to work with; especially if you come back to it and need to make changes! I'd suggest consistently sticking to always wrapping the `if`-blocks inside curly braces. 😄